src/client/pages/components/super_user/AddVideoForm.jsx

```diff
@@ -40,71 +41,82 @@ export default function AddVideoForm() {
 
   const handleSubmit = async (e) => {
     e.preventDefault();
-    const videosCollection = collection(db, 'videos');
-    const newObj = {
+    const payload = {
       ...formInput,
       createdAt: serverTimestamp(),
       updatedAt: serverTimestamp(),
     };
-    await addDoc(videosCollection, newObj);
-    resetInput();
+    const res = await post(payload, 'videos');
+    if (res.success) {
+      setFormInput(initialFormValues);
+      resetInput();
+    }
   };
 
   return (
-    <form
-      aria-label="form"
-      onSubmit={handleSubmit}
-      className='max-w-screen'
-    >
-      <FormInput
-        labelText="url"
-        type="url"
-        name="url"
-        value={formInput.url}
-        placeholder="add URL here"
-        onChange={handleInputChange}
-      />
-      <FormInput
-        labelText="videoLength"
-        type="number"
-        name="videoLength"
-        value={formInput.videoLength}
-        placeholder="add video length here"
-        onChange={handleInputChange}
-      />
-      <FormInput
-        labelText="description"
-        type="text"
-        name="description"
-        value={formInput.description}
-        placeholder="add description here"
-        onChange={handleInputChange}
-      />
-      <FormInput
-        labelText="event"
-        type="text"
-        name="event"
-        value={formInput.event}
-        placeholder="add event name here"
-        onChange={handleInputChange}
-      />
-      <FormInput
-        labelText="views"
-        type="number"
-        name="views"
-        value={formInput.views}
-        placeholder="add number of views here"
-        onChange={handleInputChange}
-      />
-      <FormInput
-        labelText="eventDate"
-        type="date"
-        name="eventDate"
-        value={formInput.eventDate}
-        placeholder="add event date here"
-        onChange={handleInputChange}
-      />
-      <button type="submit">add video</button>
-    </form>
+    <div className="w-1/3">
+      <form
+        aria-label="form"
+        onSubmit={handleSubmit}
+      >
+        <FormInput
+          labelText="title"
+          type="text"
+          name="title"
+          value={formInput.title}
+          placeholder="add video title here"
+          onChange={handleInputChange}
+        />
+        <FormInput
+          labelText="url"
+          type="url"
+          name="url"
+          value={formInput.url}
+          placeholder="add URL here"
+          onChange={handleInputChange}
+        />
+        <FormInput
+          labelText="videoLength"
+          type="number"
+          name="videoLength"
+          value={formInput.videoLength}
+          placeholder="add video length here"
+          onChange={handleInputChange}
+        />
+        <FormInput
```

Comment:
Possibly we would need to use the FormTextarea component here instead of FormInput if the description is meant to be a paragraph long. Would be easy to swap those out.